lumigator/python/mzai/backend/backend/services/datasets.py

```diff
@@ -141,11 +151,12 @@ def upload_dataset(self, dataset: UploadFile, format: DatasetFormat) -> DatasetR
             # Validate format
             validate_dataset_format(temp.name, format)
 
+            # Verify whether the dataset contains ground truth
+            has_gt = dataset_has_gt(temp.name)
+
             # Create DB record
             record = self.dataset_repo.create(
-                filename=dataset.filename,
-                format=format,
-                size=actual_size,
+                filename=dataset.filename, format=format, size=actual_size, gt=has_gt
```

Comment:
It seems obvious what it is when you're deep into it, but here's another vote to using ground_truth wherever there is gt... 😅